src/components/utils/svgA11yHidden.ts

```diff
@@ -0,0 +1,6 @@
+export const svgA11yHidden = {
```

Comment:
Not sure about standalone file and object itself cause we have only one prop needed (`aria-hidden`)

Reply:
Code itself is clear about it purpose, that could be not clear why svgs receive `aria-hidden`

Reply:
Ok, but let's name it accordingly, `a11yHiddenSvgProps`, more accent on props. And also let's rename file to `utils/svg.ts` for future svg code

Reply:
Done